cubic-server/command_parser/commands/MSPT.cpp

```diff
@@ -0,0 +1,66 @@
+#include "MSPT.hpp"
+
+#include "Chat.hpp"
+#include "Dimension.hpp"
+#include "Player.hpp"
+#include "Server.hpp"
+#include "World.hpp"
+#include "logging/logging.hpp"
+#include <vector>
+
+using namespace command_parser;
+
+void MSPT::autocomplete(UNUSED std::vector<std::string> &args, UNUSED Player *invoker) const { }
+
+void MSPT::execute(std::vector<std::string> &args, Player *invoker) const
+{
+    switch (args.size()) {
```

Comment:
Please don't use a switch case for just 3 branching.
Please also put some of that code in other functions as the execute method is getting quite big.